content/blog/sprint-review-2405.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Wazo Platform 24.05 Released
+date: 2024-04-08T08:00:00
+author: The Wazo Authors
+category: Wazo Platform
+tags: wazo-platform, development
+slug: release-review-2405
+status: published
+---
+
+Hello Wazo Platform community!
+
+Here is a short review of the Wazo Platform 24.05 release.
+
+## Bug Fixes
+
+- **Context ranges**: The context range API now return the extension ranges based on the DID length parameter of the context
+- **Tenants**: The wazo-auth /tenants list now filter using it's uuids query parameter properlly
+- **Chat**: room chat participants can no longer read messages of rooms they are not part of
+- **Chat**: room chat participants can now create rooms which include all participants of an existing room
+
+## Technical
+
+- **Parking**: A new API has been added to control parked calls
+- **Asterisk**: We are now using Asterisk 21
+
+## Ongoing Features
+
+- **wazo-provd**: The storage of wazo-provd is being rewritten to use Postgresql instead of json files on the filesystem
```

Comment:
```suggestion
## Bug Fixes

- **Context ranges**: The context range API now returns the extension ranges based on the DID length parameter of the context
- **Tenants**: The wazo-auth /tenants list now filter using it's uuids query parameter properly
- **Chat**: Room chat participants can no longer read messages of rooms they are not part of
- **Chat**: Room chat participants can now create rooms which include all participants of an existing room

## Technical

- **Parking**: A new API has been added to control parked calls
- **Asterisk**: We are now using Asterisk 21

## Ongoing Features

- **wazo-provd**: The storage of wazo-provd is being rewritten to use Postgresql instead of json files on the filesystem
```